sunpy/net/dataretriever/client.py

```diff
@@ -85,15 +85,15 @@ def response_block_properties(self):
         return s
 
     def __repr__(self):
-        return repr(type(self)) + repr(self._build_table())
+        return repr(type(self)) + repr(self.build_table())
 
     def __str__(self):
-        return str(self._build_table())
+        return str(self.build_table())
 
     def _repr_html_(self):
-        return self._build_table()._repr_html_()
+        return self.build_table()._repr_html_()
 
-    def _build_table(self):
+    def build_table(self):
```

Comment:
I think it's better to keep this as a private function.

Reply:
```python
AttributeError: 'QueryResponse' object has no attribute '_build_table'
```
This occurs if we keep it as a private function.

Reply:
I wonder why that is. In that case leave it.